common/image.c

```diff
@@ -969,9 +970,17 @@ int boot_get_ramdisk (int argc, char * const argv[], bootm_headers_t *images,
 			break;
 #endif
 		default:
-			puts ("Wrong Ramdisk Image Format\n");
-			rd_data = rd_len = rd_load = 0;
-			return 1;
+#ifdef CONFIG_SUPPORT_RAW_INITRD
```

Comment:
Does this section come from backporting an upstream commit? If so, can we take the upstream commit as-is, or at least reference it in the commit message?

Reply:
sure, since it was just a few lines and it didn't clean apply I just copy-pasted the diff. I'll reference it in the commit message.